numpy/core/__init__.py

```diff
@@ -11,7 +11,18 @@
     if envkey not in os.environ:
         os.environ[envkey] = '1'
         env_added.append(envkey)
-from . import multiarray
+
+try:
+    from . import multiarray
+except ImportError:
+    msg = """
```

Comment:
It might be usefull someday to have a `format_message` function that would remove prefixed spaces so we could make these messages look better. IIRC @pv had, or knew of, a function that did that.